docs/src/sdp/existing_configs.rst

```diff
@@ -177,18 +177,33 @@ Few-shot Learning Evaluation of Universal Representations of Speech (FLEURS)
 
    config-docs/armenian/fleurs/config
 
-English LibriSpeech (ELS)
-~~~~~~~~~~~~~~~~~~~~~~~~~
+LibriSpeech
+~~~~~~~~~~~
 
 **Dataset link:** https://openslr.org/12
+**Dataset link:** https://openslr.org/31
+
+
+**Supported configs**.
+
+* **config**:
```

Comment:
Recommend:
**config** (for processing one specific subset at a time)

Reply:
Done